response_operations_ui/exceptions/exceptions.py

```diff
@@ -5,6 +5,20 @@ def __init__(self, response):
         self.message = response.text
 
 
+class ExternalApiError(ApiError):
```

Comment:
Introduced this new `ExternalApiError` subclass to avoid a refactor of existing `ApiError` exception raises. Refactoring to have just the one class could be done separately as the existing exception should also be able to handle connection errors as well as response errors.